src/components/AlertModal/AlertModal.vue

```diff
@@ -68,33 +78,35 @@ export default {
       validator: (value) => Object.values(Modal.Alerts).includes(value),
     },
   },
-  computed: {
-    actions() {
+  setup(props, { emit }) {
+    const actions = computed(() => {
       return {
         primary: {
-          label: this.confirmLabel,
+          label: props.confirmLabel,
           handler: () => {
-            this.$emit('confirm');
+            emit('confirm');
           },
         },
         secondary: {
-          label: this.cancelLabel,
+          label: props.cancelLabel,
           handler: () => {
-            this.$emit('cancel');
+            emit('cancel');
           },
         },
       };
-    },
-    dialogProps() {
+    });
+
+    const dialogProps = computed(() => {
       return {
         actionAlignment: Modal.ActionAlignments.CENTER,
         header: Modal.Headers.TRANSPARENT,
         size: Modal.Sizes.SMALL,
-        ...this.$attrs,
+        ...props.$attrs,
```

Comment:
`$attrs` isn't available on props, instead it should be available in the context (second argument). So you'd do something like: 

` setup(props, { emit, attrs })`

and then: 
```suggestion
        ...attrs,
```